tests/integration/create_token.py

```diff
@@ -1,7 +1,11 @@
+from uuid import uuid4
+
+import time
+
 from app.views.dev_mode import generate_token, create_payload
 
 PAYLOAD = {
-    'user': "mci-integration-test",
+    'user_id': "mci-integration-test",
```

Comment:
Yeah I looked into this before. There is either a flake8 plugin or pylint plugin. We should look into that for consistency but maybe on another ticket as it wasn't introduced here and we can fix quotes across the board